pkg/logentry/stages/labels_test.go

```diff
@@ -49,6 +59,26 @@ func TestLabelsPipeline_Labels(t *testing.T) {
 	assert.Equal(t, expectedLbls, lbls)
 }
 
+func TestLabelsPipelineWithMissingKey_Labels(t *testing.T) {
+	var buf bytes.Buffer
+	w := log.NewSyncWriter(&buf)
+	logger := log.NewLogfmtLogger(w)
+	pl, err := NewPipeline(logger, loadConfig(testLabelsYaml), nil, prometheus.DefaultRegisterer)
+	if err != nil {
+		t.Fatal(err)
+	}
+	lbls := model.LabelSet{}
+	Debug = true
+	ts := time.Now()
+	entry := testLabelsLogLineWithMissingKey
+	extracted := map[string]interface{}{}
+	pl.Process(lbls, extracted, &ts, &entry)
+	expectedLog := "level=debug msg=\"failed to convert extracted label value to strin\" err=\"Can't convert <nil> to string\" type=null"
+	if !(strings.Contains(buf.String(), expectedLog)) {
+		t.Errorf("\nexpected: %s\n+actual: %s", expectedLog, buf.String())
+	}
+}
+
```

Comment:
@cyriltovena @owen-d I have added the test case to validate the log lines. May be not good way to test/assert log lines. Kindly guide me for any other better ways.
If this is OK. I will add similar tests to template, regex, output and tenant

Reply:
Looks good, however tests are failing.

Reply:
Will fix.